<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EstructuraForm</class>
 <widget class="QGroupBox" name="EstructuraForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Estructura</string>
  </property>
  <property name="title">
   <string>ESTRUCTURA</string>
  </property>
  <widget class="QLineEdit" name="cuentaEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>-10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="estPisoLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Piso:</string>
      </property>
      <property name="buddy">
       <cstring>estPisoEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="estPisoEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="estMurosLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Muros:</string>
      </property>
      <property name="buddy">
       <cstring>estMurosEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="estMurosEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="estPilaresLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Pilares:</string>
      </property>
      <property name="buddy">
       <cstring>estPilaresEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="estPilaresEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="estVigasLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Vigas:</string>
      </property>
      <property name="buddy">
       <cstring>estVigasEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="estVigasEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="estArcosLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Arcos:</string>
      </property>
      <property name="buddy">
       <cstring>estArcosEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="estArcosEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="estEscaleraLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Escalera:</string>
      </property>
      <property name="buddy">
       <cstring>estEscaleraEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="estEscaleraEdit">
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
